### DISCUSSION TITLE
Thanks Thread

### DISCUSSION THREAD

**Discussion by BigWincy** [Nimdy/Dedicated_Valheim_Server_Script#46] [General]
Just want to say thanks. This script saved me a whole mess of time and research. Since the game is still in it's infancy, there isn't much out there, especially for Linux. The procedure worked seamlessly. My 3 kids and I are having a blast playing in our world.

You've done great work. Thanks, Nimdy.

- Wincy

**Comment by Nimdy** [comment]
you are welcome!

**Comment by Nimdy** [comment]
Oh if you like it, please vote!

https://steamcommunity.com/sharedfiles/filedetails/?id=2390441423

**Reply by BigWincy** [reply]
Done.

**Comment by tkarssli** [comment]
Yes thank you @Nimdy this was so seamless and easy to follow. My friend started asking me to leave my PC on all night since I was the host so I decided to look at setting up a dedicated server. I had it up and running in ~20 mins.

**Comment by jsreynolds** [comment]
I have to add to this. I followed a couple of other recent setup how-tos across the internet for a dedicated server. Also, I was a RedHat certified engineer from the 2000 time frame, so I know may way around linux although I'm rusty. 

Oddly, no matter what I did, nothing worked. I read through your script and thought - wow, I am doing basically all these things, maybe a tweak here and there, but OK I'll try. Spun up a new Digital Ocean droplet, set up the firewall, got the script and ran it. A few min later, working server.  My kids are now having a great time playing together in their own private world. 

Thank you so much for helping out the community, and Dads like me which don't always have a full day to putz about.

**Reply by Nimdy** [reply]
Oh you are more than welcome!!

**Comment by dmackerman** [comment]
Brilliant script! It's so well done!
